controllers/storagecluster/cephobjectstoreusers.go

```diff
@@ -29,6 +31,20 @@ func (r *StorageClusterReconciler) newCephObjectStoreUserInstances(initData *ocs
 				Store:       generateNameForCephObjectStore(initData),
 			},
 		},
+		{
```

Comment:
is it safe to create such a user? any user who can access this resource could use it for previlage escalation.